examples/burial-form/index.jsx

```diff
@@ -19,12 +19,15 @@ const NoMatch = (props) => (
   </main>
 );
 
+const mapProps = (values, actions) => {
```

Comment:
do we need this to be passed in now? Or is this just for example purposes?

Reply:
I'm still passing this function just for example purposes. I could remove it, but I think by having it here, our devs will remember the `transformForSubmit` functionality exists and is something they can use.